File: src/javaapp/CashPayment.java
package javaapp;

public class CashPayment {

    private float amountPaid;

    public CashPayment(float amountPaid) {
        this.amountPaid = amountPaid;
    }

    public void processPayment(float amount) {
        System.out.println("Processing cash payment of $"+amount);
        if(amountPaid >= amount) {
            System.out.print("Payment successful! Change:" + (amountPaid - amount));
        } else {
            System.out.print("Insufficient cash. Payment failed.");
        }
    }
}

File: src/kotlinapp/CashPayment.kt
package kotlinapp

class CashPayment(private val amountPaid: Float) {
    fun processPayment(amount: Float) {
        println("Processing cash payment of $$amount")
        if (amountPaid >= amount) {
            print("Payment successful! Change:" + (amountPaid - amount))
        } else {
            print("Insufficient cash. Payment failed.")
        }
    }
}

